### ISSUE TITLE
Type card Mir is not detected

### ISSUE THREAD

**Issue by Ilyaololo** [cholojuanito/credit_card_type_detector#15]
Function `detectCCType('2202 xxxx xxxx xxxx')` returns empty list. Add please

```dart
final CardCollection _defaultCCTypes = CardCollection({
  TYPE_VISA: CreditCardType.visa(),
  TYPE_MASTERCARD: CreditCardType.mastercard(),
  TYPE_AMEX: CreditCardType.americanExpress(),
  TYPE_DISCOVER: CreditCardType.discover(),
  TYPE_DINERS_CLUB: CreditCardType.dinersClub(),
  TYPE_JCB: CreditCardType.jcb(),
  TYPE_UNIONPAY: CreditCardType.unionPay(),
  TYPE_MAESTRO: CreditCardType.maestro(),
  TYPE_ELO: CreditCardType.elo(),
  TYPE_HIPER: CreditCardType.hiper(),
  TYPE_HIPERCARD: CreditCardType.hipercard(),
  TYPE_MIR: CreditCardType.mir(),                          //  <===========  this line
});```

**Comment by deckvv** [comment]
Not needed, cause this sh*t works only in ruZZia

**Comment by cholojuanito** [comment]
Should be fixed with the forgotten line in #16
